l10n_it_ipa/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+IPA Code (IndicePA)
+===================
+
+This module adds IPA (IndicePA) field to partner
+
+Installation
```

Comment:
I think you can remove 'installation' 'configuration' and ' known issues' in this case